projects/core/src/store-finder/facade/store-finder.service.spec.ts

```diff
@@ -132,24 +131,4 @@ describe('StoreFinderService', () => {
       );
     });
   });
-
```

Comment:
Shouldn't we adjust those tests instead of removing them?

Reply:
The logic of the backend was changed, and now all functionality is provided by one endpoint instead of seperated ones. Those tests are outdated and not valid anymore.